### ISSUE TITLE
Add functionality to unmount layer filesystem

### ISSUE THREAD

**Issue by nishakm** [vmware/tern#47]
under utils/rootfs.py unmount_rootfs unmounts the mergedir only once. Have it unmount for the number of layers that were overlayed. It's ok to pass the number of layers to the function

Refer to #37 for more information

**Comment by nishakm** [comment]
@snmpboy is working on this

**Comment by nishakm** [comment]
@snmpboy can you submit your PR to the layer-debug branch rather than the master branch?

**Comment by nishakm** [comment]
Merged into layer-debug branch

**Comment by nishakm** [comment]
Resolved in https://github.com/vmware/tern/commit/1d687f09e5648898afd6829fb4328971970eb755
